src/components/home/elements/TokenRecord.vue

```diff
@@ -0,0 +1,264 @@
+                    variant="link"
+                    right>
+          <template slot="button-content">
+            <div
+              @mouseover="hoverIco"
+              @mouseout="unhoverIco">
+              <img
+                v-if="hovered"
+                src="../../../assets/imgs/icons/wallet/ic_more_hover.svg">
+              <img
+                v-if="!hovered"
+                src="../../../assets/imgs/icons/wallet/ic_more.svg">
+            </div>
+          </template>
+          <b-dropdown-item @click="showModal">Get Token Info</b-dropdown-item>
+          <b-dropdown-item @click="showIssueModal">Issue Token</b-dropdown-item>
+          <b-dropdown-item @click="showModal">Burn Token</b-dropdown-item>
+        </b-dropdown>
+      </b-col>
+    </b-row>
+    <TokenInfoModal :token-id="tokenId"
+                    :address="address"
+                    :addresses="addresses">
+    </TokenInfoModal>
+    <IssueToken :address="address"
+                :wallet-type="walletType"
+                :addresses="addresses">
+    </IssueToken>
+  </b-container>
+</template>
+
+<script>
+// import TxInfoModal from './TxInfoModal'
+// import base58 from '@/libs/base58'
+// import converters from '@/libs/converters'
+// import { VSYS_PRECISION } from '@/constants'
+// import crypto from '@/utils/crypto'
+// import CancelLease from '../modals/CancelLease'
+// import { PAYMENT_TX, LEASE_TX, CANCEL_LEASE_TX } from '../../../constants'
+import browser from '../../../utils/browser'
+import TokenInfoModal from './TokenInfoModal'
+import IssueToken from './IssueToken'
+export default {
+    name: 'TokenRecord',
+    components: { TokenInfoModal, IssueToken },
+    data: function() {
+        return {
+            differenceHeight: 0,
+            heightStatus: false,
+            hovered: false,
+            cancelTime: 0,
+            showCancelDetails: false
+        }
+    },
+    props: {
+        txRecord: {
+            type: Object,
+            default: function() {
+                return {
+                    txType: '',
+                    txAddress: '',
+                    txTime: '',
+                    txAmount: 0,
+                    txAttachment: ''
+                }
+            }
+        },
+        address: {
+            type: String,
+            default: ''
+        },
+        isCanceled: {
+            type: Boolean,
+            default: false
+        },
+        walletType: {
+            type: String,
+            default: 'hot wallet'
+        },
+        addresses: {
+            type: Object,
+            default: function() {},
+            require: true
+        }
+    },
+    computed: {
+        txAddressShow() {
+            if (this.txAddress) {
+                const addrChars = this.txAddress.split('')
+                addrChars.splice(6, 23, '******')
+                return addrChars.join('')
+            }
+        },
+        txTitle() {
+            return this.address
```

Comment:
This should be tokenID or tokenTicker. It should be load from local storage. If not implemented, just fill with a hard code string first.